test/Microsoft.IdentityModel.Protocols.OpenIdConnect.Tests/OpenIdConnectConfigurationRetrieverTests.cs

```diff
@@ -28,7 +30,15 @@ public async Task FromNetwork()
         [Fact]
         public async Task FromFile()
         {
-            var context = new CompareContext();
+            var context = new CompareContext
+            {
+                PropertiesToIgnoreWhenComparing = new Dictionary<Type, List<string>>
+                {
+                    // If the objects being compared are created from the same string and they are equal, the string itself can be ignored.
```

Comment:
I think we should still compare the `JsonData` to verify this assumption against regressions - is the original string passed into the constructor actually assigned to the property.

Reply:
> is the original string passed into the constructor actually assigned to the property

This is verified by `JsonWebKeySetTests`